tests/test_tensorflow_bigquery.py

```diff
@@ -9,6 +9,11 @@ class TestTensorflowBigQuery(unittest.TestCase):
     # Some versions of bigquery crashed tensorflow, add this test to make sure that doesn't happen.
     # python -c "from google.cloud import bigquery; import tensorflow". This flow is common because bigquery is imported in kaggle_gcp.py
     # which is loaded at startup.
-    def test_addition(self):        
+
+    def test_matrix_addition(self):        
         result = tf.add([1, 2], [3, 4])
         self.assertEqual([2], result.shape)
```

Comment:
asserting on the value returned by `.numpy()` will indirectly assert the shape. Should we consolidate these into one matrix addition test asserting on `.numpy()`?